packages/webapp/src/components/Finances/AddExpense/ExpenseItemInputs.jsx

```diff
@@ -22,7 +22,7 @@ import { DATE, NOTE, VALUE } from './constants';
 import styles from './styles.module.scss';
 import { useCurrencySymbol } from '../../../containers/hooks/useCurrencySymbol';
 
-export default function ExpenseItemInputs({ register, onRemove, getErrors }) {
+export default function ExpenseItemInputs({ register, onRemove, getErrors, itemsQuantity }) {
```

Comment:
Thank you for clarifying the requirement in the ticket!

Looking at this component as a stand-alone component, it's not clear what `itemsQuantity` refers to. I think something like isRemovable would make more sense.